### DISCUSSION TITLE
Learning K3s, starting with single-node cluster. Any reason I couldn't use Longhorn with a single-node cluster?

### DISCUSSION THREAD

**Discussion by loganmarchione** [longhorn/longhorn#4668] [Q&A] [answered]
I'm learning K3s, and wanted to start with a single-node cluster. However, I wanted to be able to expand to multiple nodes in the future, so I thought I could install Longhorn now, on a single node, and be able to expand in the future if I add more nodes. Is there any reason I shouldn't do this?

**Comment by c3y1huang** [comment, marked as answer]
Longhorn won't be able to support HA with a single node. This means the cluster is potentially at risk of having service interruption or losing data if something goes wrong with that node.

Ref https://longhorn.io/docs/1.3.1/best-practices/

**Reply by loganmarchione** [reply]
@c3y1huang thanks for the reply. I'm willing to accept that risk since it's just a cluster for learning.

**Comment by fonsc** [comment]
Thanks for raising this discussion! I'm on the same boat, learning k3s on a single-node cluster. What attracted me in Longhorn was not HA but easy **volume encryption** and **snapshots/backups**. I could not find straightforward way to achieve the same without Longhorn. Any tips about alternative tools that would be better suited for those use cases?

**Reply by PhanLe1010** [reply]
non on top of my head. [local path provisioner](https://github.com/rancher/local-path-provisioner) might serve some of your use cases. But it doesn't have `volume encryption` and `snapshots/backups`
